Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -134,4 +149,21 @@ class EventNotificationsManager {
             self.sendNotifications(forEvents: foundEvents)
         })
     }
+
+    fileprivate func isUnsent(event: Event) -> Bool {
+        guard let placeEvents = sentNotifications[event.placeId] else {
+                return true
+        }
+        return !placeEvents.contains(event.description)
+    }
+
+    fileprivate func markAsSent(event: Event) {
+        var sent = sentNotifications
+        if sent.keys.contains(event.placeId) {
+            sent[event.placeId]?.append(event.description)
+        } else {
+            sent[event.placeId] = [event.description]
```

Comment:
nit: Not sure we can store in UserDefaults but this would be more efficient as a set (at least, as these grow larger and hashing time is more efficient than array search).